adoc/Release-Notes.adoc

```diff
@@ -107,83 +179,54 @@ For information about deploying and administering {product}, see the product man
 
 [IMPORTANT]
 ====
-In order to deploy {product} 1.5 or upgrade from {product} 1.4.1 in an HA configuration, you will first need to scale the `mysql` role down to 1 instance to be able to migrate from `cf-mysql` to `PXC`. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment.
+In order to deploy {product} 1.5 or upgrade from {product} 1.4.1 in an HA configuration, you will first need to start the `mysql` role with 1 instance to be able to migrate from `cf-mysql` to `PXC`. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment.
```

Comment:
why is it talking about 1.4.1. It should be 1.50 to 1.5.1

Reply:
This PR also incorporates the revised 1.4.1 to 1.5.0 upgrade instructions, merged from another PR (84) since we're working on both simultaneously. It's possible we have to go further back and update things, so this is a combined update.

Reply:
This PR also incorporates the revised 1.4.1 to 1.5.0 upgrade instructions, merged from another PR (84) since we're working on both simultaneously. It's possible we have to go further back and update things, so this is a combined update.